code/client/client.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include <yarp/os/all.h>
+
+using namespace std;
+using namespace yarp::os;
+
+int main(int argc, char * argv[])
+{
+	if(argc<2){
+		cout << "Please supply a port name for client" << endl;
+		return 1;
+	}
+
+	const char *charName = argv[1];
+
+    Network yarp;	//initialize yarp
+    Semaphore sem;
+    Port portP;
+    portP.open(charName);
+    
```

Comment:
I thought I had to use yarp::os::port class only because the prerequisites of assignment said to use it.
I changed the ports to RPC ports.